ground/gcs/src/plugins/pathplanner/modeluavoproxy.cpp

```diff
@@ -124,6 +132,25 @@ bool ModelUavoProxy::modelToObjects()
             m_eventloop.exec();
         }
     }
+
+    for (; x < instances; x++) {
+        Waypoint *wp = Waypoint::GetInstance(objManager, x);
+                // shadows wp above
```

Comment:
It's a for loop without initializing x first.  e.g. continue counting up from where we were.